edx_proctoring/views.py

```diff
@@ -496,6 +499,61 @@ def get(self, request):
         elif relevant_attempt:
             data['onboarding_status'] = relevant_attempt.status
 
+        if waffle.switch_is_active(ONBOARDING_PROFILE_API):
+            attempt_data = None
+            try:
+                attempt_data = backend.get_onboarding_profile_info(course_id=course_id, user_id=user.id)
+            except BackendProviderOnboardingProfilesException as exc:
+                log_message = (
+                    'Failed to use backend onboarding status API endpoint for user_id={user_id}'
```

Comment:
#NIT Please add to the warning message saying, the `onboarding status will be determined by user's onboarding exam attempts` or something like that?